cni/nnsgrpcclient/nnsgrpcclient_test.go

```diff
@@ -0,0 +1,117 @@
+package nnsgrpcclient
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-container-networking/test/nnsmockserver"
+	"os"
+	"strings"
+	"testing"
+	"time"
+)
+
+var mockserver *nnsmockserver.NnsMockServer
+
+func TestMain(m *testing.M) {
+	setup()
+	code := m.Run()
+	teardown()
+	os.Exit(code)
+}
+
+func setup() {
+	mockserver  = nnsmockserver.NewNnsMockServer()
+	go  mockserver.StartGrpcServer(nnsPort)
+	fmt.Printf("mock nns server started\n")
+}
+
+func teardown() {
+	mockserver.StopGrpcServer()
+	fmt.Printf("mock nns server stopped\n")
+}
+
+// CNI ADD to add container to network
+func TestAddContainerNetworking(t *testing.T) {
+
+	client := &NnsGrpcClient{}
+	if err := client.AddContainerNetworking(
+		 context.Background(),
+		"sf_8e9961f4-5b4f-4b3c-a9ae-c3294b0d9681",
+		"testnwspace"); err != nil {
+       t.Fatalf("TestAddContainerNetworking failed: %v", err)
```

Comment:
formatting seems off. if you are using goland, you can enable file watchers and goimports (like gofmt but with added functionality)

Reply:
ran gofmt on this file.